src/tools/crossgen2/crossgen2/crossgen2.csproj

```diff
@@ -11,7 +11,6 @@
     <!-- We're binplacing these into an existing publish layout so that F5 build in VS updates
```

Comment:
Are these comments still relevant with `OutputPath` property deleted?

Reply:
I think, based on my understanding, it was more about the .json files that were generated by the build, which were messing up the VS F5 experience.

Reply:
The json files generated by the build were overwriting the json files publish created and made the published layout no longer work. This can all be deleted if you're no longer binplacing them in the published layout.

Reply:
The runtimeconfig we'd be generating here wouldn't be useful for a framework dependent execution. Keeping it the way it is.